### PR TITLE
RN(migration tool): update changelog for 2.40.x

### PR DESCRIPTION
**Pull request by educhastenier**

### CODE REVIEW

**Comment by educhastenier** [comment]
I removed the pre-7.6.0 versions, as they are not supported anymore.
:question: I hope it's ok for you...

File: md/migration-tool.md [hunk 1]
```diff
@@ -5,6 +5,20 @@ to the one delivered with the latest version of Bonita.
 This is due to the fact that improvements in any version of the Migration Tool can affect all supported version of Bonita.
 This change log must be read before [migrating to a newer version of Bonita](migrate-from-an-earlier-version-of-bonita-bpm.md).
 
+## 2.40.3 - May. 27th, 2019 ??
```
Comment (tbouffard) [inline md/migration-tool.md]:
```suggestion
## 2.40.3 - May. 27th, 2019 (Bonita 7.8.4)
```

File: md/migration-tool.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ to the one delivered with the latest version of Bonita.
...
+This version of the migration tool migrates Bonita up to version 7.8.4.
+* BS-19346 reduce disk space used when migrating (archived) contract data in version 7.7.0
+
+## 2.40.2 - May. 23rd, 2019 ??
```
Comment (tbouffard) [inline md/migration-tool.md]:
```suggestion
## 2.40.2 - May. 23rd, 2019 (Bonita 7.8.4)
```

File: md/migration-tool.md [hunk 3]
```diff
@@ -5,6 +5,20 @@ to the one delivered with the latest version of Bonita.
...
+This version of the migration tool migrates Bonita up to version 7.8.4.
+* Configuration: make datasource pool size configurable
+* Logs: introduce timestamp in the log files
+* BS-19267 improve performance when migrating (archived) contract data in version 7.7.0
+
+## 2.40.1 - April. xxth, 2019 ??
```
Comment (tbouffard) [inline md/migration-tool.md]:
```suggestion
## 2.40.1 - May. 17th, 2019 (Bonita 7.8.4)
```

**Review by tbouffard — CHANGES_REQUESTED** [review]

**Review by tbouffard — APPROVED** [review]

**Pull request merged**